conftest.py

```diff
@@ -199,14 +198,10 @@ def hashing(request):
             with open(request.cls.answer_file) as fd:
                 request.cls.saved_hashes = yaml.safe_load(fd)
         except FileNotFoundError:
-            # On travis and appveyor only a minimal set of answer tests are
-            # run, which means that, for most answer tests, there won't be
-            # an existing answer file when comparing. There is currently no
-            # list of the minimal answer tests, so they can't be marked.
-            # As such, if we're comparing and the file of saved hashes isn't
-            # found, we just skip the test. We do the skip before the test
-            # is run to save time
-            pytest.skip("Answer file not found.")
+            module_filename = f"{request.function.__module__.replace('.', os.sep)}.py"
```

Comment:
Ah ! Upon second look I wonder if there's a difference with the simpler
`request.function.__file__ˋ
?

Reply:
Believe me...

```
>>> print(request.function.__module__)
yt.frontends.gdf.tests.test_outputs
>>> print(request.function.__file__
... )
Traceback (most recent call last):
  File "/home/xarth/codes/xarthisius/yt/conftest.py", line 198, in hashing
    with open(request.cls.answer_file) as fd:
FileNotFoundError: [Errno 2] No such file or directory: '/home/xarth/codes/xarthisius/yt/answer-store/TestGDF_000.yaml'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/code.py", line 90, in runcode
    exec(code, self.locals)
  File "<console>", line 1, in <module>
AttributeError: 'function' object has no attribute '__file__'
```

I've tried.

Reply:
🙈